README.md

```diff
@@ -3297,11 +3297,23 @@ http_basic do |username, password|
 end
 ```
 
-```ruby
-http_digest({ realm: 'Test Api', opaque: 'app secret' }) do |username|
-  # lookup the user's password here
-end
-```
+For Digest Auth
+
+* Compare with the pure password
```

Comment:
`pure` -> `clear text`

"Digest auth supports clear-text passwords and password hashes."

... then just provide the two examples without any other text?

Reply:
I agree with you, it sounds better =)